composer.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "kuzzleio/kuzzle-sdk",
-  "version": "3.0.3",
+  "version": "3.0.4",
```

Comment:
Why only minor update ?

Reply:
I don't know why but I did not change this field.